metals/src/main/scala/scala/meta/internal/metals/Compilers.scala

```diff
@@ -373,6 +373,12 @@ class Compilers(
                     toDebugCompletionItem(
                       _,
                       adjustStart,
+                      Position.Range(
+                        input,
+                        // account for the added ;
+                        lineStart + 1,
+                        lineStart + 1,
```

Comment:
why isn't that the range of the currently inserted word?

Reply:
The completions don't use range here, only start and length, so no use setting the end.

Reply:
Actually they use end I think, changing end to 
`lineStart + expressionOffset(expressionText, indentation) + 1` fixes tests

Reply:
Thanks done! I always changed the input to correspond to the modified source.